src/Modal/index.js

```diff
@@ -502,9 +502,10 @@ export class PureModal extends React.PureComponent<Props & ThemeProps, State> {
   };
 
   handleClickOutside = (ev: MouseEvent) => {
-    const { onClose } = this.props;
+    const { onClose, preventOverlayClose } = this.props;
```

Comment:
Plz specify default value `false` for `preventOverlayClose`, otherwise it won't be possible to close with with clicking in the overlay 🙁 
'cause `undefined === false` is `false`